src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -413,8 +413,6 @@ julia> ngens(ideal_of_linear_relations(p2))
 """
 @attr MPolyIdeal function ideal_of_linear_relations(v::NormalToricVarietyType)
     R, _ = polynomial_ring(coefficient_ring(v), coordinate_names(v))
```

Comment:
This needs the graded polynomial ring (with standard grading, i.e. all variable receive degree 1), so for instance:
```suggestion
@attr MPolyIdeal function ideal_of_linear_relations(v::NormalToricVarietyType)
    R, _ = graded_polynomial_ring(coefficient_ring(v), coordinate_names(v), cached = false)
```
Also needs `cached = false`, unless I am mistaken.

Reply:
OK, so that means the current code is buggy?

Reply:
(because it calls `grade` and then throws away the result)

Reply:
The grading is currently never used, or so I believe. I (tried to) add it, in order to mimic the theory, which says "the ideal of linear relations sits in a graded polynomial ring...".

Reply:
OK, changed this PR now